demo/pages/export/index.js

```diff
@@ -1,7 +1,24 @@
-Controller.$inject = ['$http'];
-export default function Controller($http) {
+import fileSaver from 'file-saver';
+import XLSX from 'xlsx';
+import jsPDF from 'jspdf';
+import autotable from 'jspdf-autotable';
+
+Controller.$inject = ['$http', 'qgrid'];
+export default function Controller($http, qgrid) {
 	this.rows = [];
 
+	this.model = qgrid.model();
+	this.model.plugin({
+		imports: {
```

Comment:
I don't think it's a good idea to require external dependencies like this. Why not use commonjs?